pkgs/by-name/fr/frequest/package.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, fetchFromGitHub, qt5 }:
+
+let
+  version = "1.2a";
+
+  src = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "FRequest";
+    repo = "FRequest";
+    rev = "v${version}";
+    hash = "sha256-fdn3MK5GWBOhJjpMtRaytO9EsVzz6KJknDhqWtAyXCc=";
+  };
+
+  commonLibsSrc = fetchFromGitHub {
+    owner = "fabiobento512";
+    name = "CommonLibs";
```

Comment:
```suggestion
```

Reply:
@SuperSandro2000 , I don't think this is possible with the current setup.

Basically the upstream compilation process requires the following directory structure:

```
FREquest
  // contents of the FRequest repository
CommonLibs
  // contents of the common lib repository
CommonUtils
  // contents of the common utils repository
```

I believe `FRequest` literally references `../CommonLibs`.

It seems that adding `name` to the sources tells the unpacker to give me this exact directory structure that is needed. Setting `sourceRoot` (apart from not failing due to multiple sources) changes directory exactly to the `FRequest` where I need to build the `.pro` file.

Reply:
Then we keep it :)